Sources/Url.php

```diff
@@ -443,6 +443,8 @@ public function proxied(): object
 			return $proxied;
 		}
 
+		$parsed = $this->parse();
+
```

Comment:
There is a mistake in this method, but this isn't the right solution. Instead, all occurrences of `$parsed` in the rest of the method should be replaced with `$proxied`.

Reply:
Ok, I was thinking that but wasn't sure.  I checked 2.1 and it was using `$parsed` as well, so I assumed that is what we meant.